.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: new feature
```

Comment:
We have 13 "new feature" issues and 29 "enhancement" issues.
Is there really a meaningful separation or just by accident?
If the latter, those should probably be consolidated and the survivor used here.

Reply:
It's a bit of semantics, new feature would be something that isn't supported at the moment. Enhancement would be to take an existing feature and enhance it. The line is a bit blurry, so for now I would stick with `new feature` for the template.